benchmarks/src/jmh/java/io/grpc/benchmarks/netty/AbstractBenchmark.java

```diff
@@ -139,8 +140,14 @@ public void setup(ExecutorType clientExecutor,
     }
 
     // Always use a different worker group from the client.
-    serverBuilder.workerEventLoopGroup(new NioEventLoopGroup());
+    serverBuilder.workerEventLoopGroup(new NioEventLoopGroup(1, Executors.newSingleThreadExecutor()));
```

Comment:
Not really, remnant from checking to see if sharing FJ pool between client and server had eny perf effect. Reverted.